### PR TITLE
added pass/password fix

### PR DESCRIPTION
**Pull request by madushan1000**

### CODE REVIEW

**Comment by mnmtanish** [comment]
Shall we use one of `pass/password` instead of both?

**Comment by madushan1000** [comment]
Changed

File: lib/modules/utils.js [hunk 1]
```diff
@@ -49,13 +49,13 @@ export function runSSHCommand(info, command) {
     var ssh = {
       host: info.host,
       port: (info.opts && info.opts.port || 22),
-      username: info.user
+      username: info.username
     };
 
     if (info.pem) {
       ssh.privateKey = fs.readFileSync(path.resolve(info.pem), 'utf8');
     } else if (info.pass) {
```
Comment (mnmtanish) [inline lib/modules/utils.js]: I think it should be

````
} else if (info.password) {
```

**Comment by madushan1000** [comment]
yup, you're right.

**Comment by mnmtanish** [comment]
@arunoda can you check this out?

Changing `user/pass` fields in mup.js to `username/password` because everyone's keep using it that way.  And, **arunoda/meteor-up** used `username/password` so I guess it'll be more familiar.

**Comment by arunoda** [comment]
Great.

File: README.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ module.exports = {
   servers: {
     one: {
       host: '1.2.3.4',
-      user: 'root'
+      username: 'root'
```
Comment (ramirez530) [inline README.md:33]: Root

**Pull request merged**